os-hardening/templates/profile.conf.tmpl

```diff
@@ -1,4 +1,4 @@
-# {{ pillar['message_do_not_modify'] }}
+# {{ salt['pillar.get']('message_do_not_modify','MANAGED BY SALT - DO NOT MODIFY') }}
```

Comment:
@askholme Can you add an example of how ``message_do_not_modify`` can be set from the pillar in ``pillar.example``